dist/types.js

```diff
@@ -373,3 +373,10 @@ exports.basicTypes = {
 if (typeof Buffer !== "undefined") {
     exports.basicTypes.Buffer = new BasicType((v) => Buffer.isBuffer(v), "is not a Buffer");
 }
+// Support typed arrays of various flavors
+for (const array of [Int8Array, Uint8Array, Uint8ClampedArray, Int16Array, Uint16Array,
+    Int32Array, Uint32Array, Float32Array, Float64Array, ArrayBuffer]) {
+    if (typeof array !== "undefined") {
```

Comment:
Actually this isn't needed. If the symbol is not available in JS, then `typeof Foo` would let us detect it (as for Buffer above, which isn't normally available as a global in browsers); but as long as the for loop in the line above doesn't fail, this symbol is available, so there isn't really a point to this typeof check.